.cirrus.yml

```diff
@@ -1,8 +1,14 @@
 freebsd_instance:
-  image: freebsd-11-3-stable-amd64-v20190801
+  image_family: freebsd-13-0-snap # 13.0 boots way faster
```

Comment:
How can I tell which GCE images are included in an `image_family`?  And I don't want to move CI to 13.0.  The minimum supported version is 11.3.  13.0 isn't even stable.  Do you know why 13-0-snap boots faster than 11-3-stable?

Reply:
Got it. The family thing just refers to the latest build of this version. So it's moving. Will revert to `freebsd-11-3-stable-amd64-v20190801`.

It boots faster from my experiments. I can see it usually boots in under a minute.